migrations/20161202144619_add_tag_everyone_can_create.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddTagEveryoneCanCreate extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * More information on this method is available here:
+     * http://docs.phinx.org/en/latest/migrations.html#the-change-method
+     *
+     * Uncomment this method if you would like to use it.
+     *
+    public function change()
+    {
+    }
+    */
+    
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $this->table('tags')
+            ->addColumn('everyone_can_create', 'boolean', [
```

Comment:
Hmm, everyone_can_create comes from the survey-form, where that is used. I could change it to 'everyone' in the directive instead, to make it more general?